scheduler.asd

```diff
@@ -13,3 +13,10 @@
   :components ((:file "scheduler")
                (:static-file "README.md")
                (:static-file "LICENSE")))
+
+(defsystem "scheduler/sqlite"
+  :depends-on ("scheduler"
+               "marshal"
```

Comment:
please remove marshal in favor of functions `serialize-task` and `deserialize-task`; implementing these should be rather straightforward. Marshaling local time goes like:

> (local-time:format-timestring nil timestamp)
> (local-time:parse-timestring *)

Reply:
Writing a serialiser is a non-trivial development. It will require much of the work already done in a serialiser library. Is there a reason not to use Marshal? Is another an option?

Reply:
Minimizing dependencies is one, another is that I've looked at marshal and the code quality is dubious.

It is not about serializing arbitrary objects, all you need to serialize are the source-entry (it is already a string!), last-extecution and next-execution.

I need to go now, but if you still insist on a library then I'll give it another thought.

Reply:
Done.